package.json

```diff
@@ -8,7 +8,11 @@
     "new-template": "node _helpers/add-template.js",
     "add-dependency": "npm install $npm_config_package --no-package-lock --prefix $npm_config_template && npm install --save-dev $npm_config_package",
     "env-manifest": "node _helpers/generate-env-variable-manifest",
-    "ci:generate-manifest": "npm run env-manifest -- --write"
+    "ci:generate-manifest": "npm run env-manifest -- --write",
+    "format": "prettier --write .",
+    "format:staged": "pretty-quick --staged",
```

Comment:
In the Serverless Toolkit we use lint-staged and husky instead. The benefit is that everything is configured in the `package.json` and we don't have to create any sh files

https://github.com/twilio-labs/serverless-toolkit/blob/main/package.json#L38-L49

Reply:
From a quick read through of the husky 6 docs, it seems that using bash scripts is the way forward. Which begs the question, why even have husky if you can just add a git commit hook yourself?

Reply:
@philnash I just use `husky` as a Git hook installer at this point -- a way to guarantee Git gets configured with our hooks during an `npm install` regardless of where it's running. I can argue for it from a developer familiarity standpoint; many of our contributors will know how to use it (or where to go for docs if they don't), and `husky.sh`'s conventions might be expected by our developers or their tooling.